101-azure-search-create/azuredeploy.json

```diff
@@ -30,7 +32,7 @@
       "maxValue": 12,
       "defaultValue": 1,
       "metadata": {
-        "description": "Replicas distribute search workloads across the service. You need 2 or more to support high availability (applies to Basic and Standard only)."
+        "description": "Replicas distribute search workloads across the service. You need two or more to support high availability (applies to Basic and Standard only)."
```

Comment:
Should we be more explicit about what "high availability" means here or are we comfortable with the other documentation clarifying that?

Reply:
Is this better? The revision mentions that "high availability" applies to queries:   "Replicas distribute search workloads across the service. You need two or more to support high availability for query workloads (applies to Basic and Standard only)".

Reply:
High availability means different things to different people. We provide guarantees around our quality of service based on the number of replicas so we may as well be explicit with that. I would say something more along the lines of "if you need read SLA then you need to have at least 2 replicas, if you need read/write SLA then you need to have at least 3 replicas."